.github/renovate.json5

```diff
@@ -37,6 +37,9 @@
       // TODO(ROX-27352): "after 3am and before 7am",
       "at any time",
     ],
+    "automerge": true,
```

Comment:
> I added `Red Hat Konflux` app in the bypass list for `main` branch.

The bypass list for requiring reviews? 
Is building the tasks a required check on PRs?

Reply:
> The bypass list for requiring reviews?

Yes. As I understand it, it will give Renovate permissions to merge to `main` without approvals from us. From docs:

> If you have protected your base branch with a list of allowed committers, and Renovate is not on that list, then naturally automerge can't work.

There might be an issue due to Codeowners settings. Let's see how it pans out.
Docs say:

> Depending on the platform, having a CODEOWNERS file could block automerging, because it means a code owner must review the PR.

----

>  Is building the tasks a required check on PRs?

According to the [Renovate docs](https://docs.renovatebot.com/key-concepts/automerge/#absence-of-tests), it should wait for CI checks.

> By default, Renovate will not automerge until it sees passing status checks / check runs for the branch. If you have no tests but still want Renovate to automerge, you need to add "ignoreTests": true to your configuration.

How it waits, how are we going to be notified, whether it will succeed automerging - all this is unknown to me because I haven tried. However, it's relatively safe to enable in this repo because we pin tasks bundle image by digest in the places were we use it and also because its image updates in tasks, i.e. quite localized.

What I mean is that I'm not sure this PR alone will be sufficient but that's ok, I'll keep the task open until things work to our liking.